app/about_us/our_values/page.module.css

```diff
@@ -0,0 +1,41 @@
+.ourValues p {
+  font-size: 16px;
+}
+
+.container {
+  display: grid;
+  grid-template-columns: repeat(2, 1fr);
+  gap: 20px;
+  margin-left: 40px;
+  padding: 0 20px;
+}
+
+.value {
+  margin-right: 40px;
+}
+
+.value hr {
+  margin: 20px 0 0 0;
+}
+
+.value h2 {
+  font-size: 36px;
```

Comment:
It is a good practice to use the same font-sizes for all the common elements, so this could be living in the globals.css file

Reply:
I made some changes during maintaining the heading convention and now this section also have specific margin for the specific element, so I am not moving it to global.css. because by doing so the margin parameter is stopped implementing to the particular element.